kaidan.qrc

```diff
@@ -3,5 +3,6 @@
 		<file alias="main.qml">src/qml/main.qml</file>
 		<file alias="RosterPage.qml">src/qml/RosterPage.qml</file>
 		<file alias="LoginPage.qml">src/qml/LoginPage.qml</file>
+		<file alias="banner.png">data/images/banner.png</file>
```

Comment:
I think we should split media and code, better name the alias "data/banner.png".